### ISSUE TITLE
Unable to pull in comic data

### ISSUE THREAD

**Issue by Shredder5262** [mylar3/mylar3#478]
this is a new install and I believe i have it set up correctly, It seems That no comic data is loading, even after adding only a series with one comic in it data just clocks from being added, and I think this is just pulling in metadata at this point, not even attempting to download yet 
I guess the question is, how long does this normally take to import comic data when trying to add a listed comic series? 

Screenshot:
https://i.imgur.com/eiQUUy7.png

`
02-Oct-2020 17:18:13 - INFO    :: mylar.addbyid.525 : CP Server Thread-11 : Attempting to add directly by ComicVineID: 125827
02-Oct-2020 17:18:16 - INFO    :: mylar.addComictoDB.131 : Thread-17 : Now adding/updating: Artemis & the Assassin
02-Oct-2020 17:18:16 - INFO    :: mylar.addComictoDB.184 : Thread-17 : Sucessfully retrieved details for Artemis & the Assassin
02-Oct-2020 17:18:16 - INFO    :: mylar.addComictoDB.190 : Thread-17 : Previous version detected as None - seeing if update required
02-Oct-2020 17:18:16 - ERROR   :: mylar.excepthook.315 : Thread-17 : Uncaught exception: Traceback (most recent call last):
  File "C:\Mylar\mylar\logger.py", line 337, in new_run
    old_run(*args, **kwargs)
  File "C:\Users\downloader\AppData\Local\Programs\Python\Python37-32\lib\threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Mylar\mylar\importer.py", line 223, in addComictoDB
    comvalues = dothedew.folder_create()
  File "C:\Mylar\mylar\filers.py", line 213, in folder_create
    ddir = pathlib.PurePath(mylar.CONFIG.DESTINATION_DIR)
  File "C:\Mylar\lib\pathlib.py", line 585, in __new__
    return cls._from_parts(args)
  File "C:\Mylar\lib\pathlib.py", line 613, in _from_parts
    drv, root, parts = self._parse_args(args)
  File "C:\Mylar\lib\pathlib.py", line 605, in _parse_args
    % type(a))
TypeError: argument should be a path or str object, not <class 'NoneType'>

02-Oct-2020 17:18:29 - WARNING :: mylar.havetotals.1420 : CP Server Thread-8 : Comic ID: 125827 has not finished loading. Nulling some values so things display properly until they can populate.
02-Oct-2020 17:18:29 - WARNING :: mylar.havetotals.1420 : CP Server Thread-8 : Comic ID: 126144 has not finished loading. Nulling some values so things display properly until they can populate.
02-Oct-2020 17:18:29 - WARNING :: mylar.havetotals.1420 : CP Server Thread-8 : Comic ID: 796 has not finished loading. Nulling some values so things display properly until they can populate.
02-Oct-2020 17:18:48 - INFO    :: mylar.add2futurewatchlist.1978 : CP Server Thread-4 : 2020-09-30
02-Oct-2020 17:18:48 - INFO    :: mylar.add2futurewatchlist.1985 : CP Server Thread-4 : Adding Victor's Crowley Hatchet: Halloween Tales II # 1 [American Mythology] to future upcoming watchlist
02-Oct-2020 17:18:53 - WARNING :: mylar.havetotals.1420 : CP Server Thread-6 : Comic ID: 125827 has not finished loading. Nulling some values so things display properly until they can populate.
02-Oct-2020 17:18:53 - WARNING :: mylar.havetotals.1420 : CP Server Thread-6 : Comic ID: 126144 has not finished loading. Nulling some values so things display properly until they can populate.
02-Oct-2020 17:18:53 - WARNING :: mylar.havetotals.1420 : CP Server Thread-6 : Comic ID: 796 has not finished loading. Nulling some values so things display properly until they can populate.
`

**Comment by barbequesauce** [comment]
What is destination_dir set to in your config.ini? Make sure the slashes point the right way (ie Windows not Linux-style)

**Comment by evilhero** [comment]
You don't have a Comic Location set in your config - so Mylar doesn't know where to move your imports. 

Add a location, update your config and then try it again from the import results screen (no need to rescan everything).

If you don't want to move anything than make sure you enable the **Set import paths to series location paths** option prior to scanning (if you set it again, you'd have to flush your existing import and then rescan so it can pick the right directories)

**Comment by Shredder5262** [comment]
Wow, that was a fast response...I think i got it, I'm pulling in comic data now and that seems to be working on a somewhat consistent basis now...trying to get a successful download now.....feeling my way through the interface. We'll see how it goes, thanks for the assist

**Comment by evilhero** [comment]
Nzbgeek / nzb Planet /any other indexer that's newznab based (they usually say) you would use the newznab option (Enable Newznab) and add them in manually that way.

If you add a  comic to the wanted list, mylar will search for it immediately in a queue like way of first in, first out. If it doesn't find anything, then it fires off a search every 6hrs by default for the issues on your wanted list. 

Unless you do a Force Check, then that will perform an immediate search for everything in your wanted list that's a Tier 1 (as shown on the Wanted tab). Tier 2 items don't get searched for  automatically, as with rss feeds enabled things will be continually monitored - so Tier 2 items you'd have to manually search for (at least for the time being).

If you want to do an immediate search (unless there's a backlog in the search queue, then it gets tacked on to the end of the queue), you can also use the magnifying glass on the series detail page for a given series on your watchlist. This will fire off the search, but it will not mark the issue as Wanted, and thereby won't be searched automatically when the 6hr delay/force check search fires off.

There are other things probably, but the most important one is getting your indexers entered in correctly. Enable the DDL option for direct  downloads, and maybe enable experimental if you're already using nzb providers (experimental searches use nzbindex).

Hope that helps!

**Comment by Shredder5262** [comment]
That does, immensely, thank you! I have made it to the point where i can say it is working and I am getting the output that i want, however results are very few and far between, I have 3 providers added, DDL and experimental enabled....out of the 780+ comics I have added to search for, I was able to grab 6 results.....this may be a forbidden question, but is there anything i can do beyond this to increase results? is there a good provider out there in regards to comics that i can increase results?
